easybuild/easyconfigs/t/TurboVNC/TurboVNC-2.2.3-GCCcore-8.2.0.eb

```diff
@@ -20,12 +24,19 @@ builddependencies = [
 
 dependencies = [
     ('X11', '20190311'),
+    ('pixman', '0.38.0'),
     ('libjpeg-turbo', '2.0.2'),
+    ('libglvnd', '1.2.0'),
 ]
 
 osdependencies = [('pam-devel', 'libpam0g-dev')]
 
-configopts = "-DTVNC_BUILDJAVA=0"
+configopts = "-DTVNC_BUILDJAVA=0 -DTVNC_SYSTEMX11=1 "
+configopts += "-DX11_X11_LIB=$EBROOTX11/lib/libX11.so -DX11_Xau_LIB=$EBROOTX11/lib/libXau.so "
+configopts += "-DX11_SM_LIB=$EBROOTX11/lib/libSM.so -DX11_ICE_LIB=$EBROOTX11/lib/libICE.so "
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libXext.so -DX11_ICE_LIB=$EBROOTX11/lib/libICE.so "
+configopts += "-DX11_Pixman_LIB=$EBROOTPIXMAN/lib/libpixman-1.so.0 "
+configopts += "-DOPENGL_gl_LIBRARY=$EBROOTLIBGLVND/lib/libGL.so "
```

Comment:
Hmm, overlooked that, thanks.

Might as well specify `-DOPENGL_opengl_LIBRARY` and `-DOPENGL_glx_LIBRARY` too, since they're both provided by `libglvnd`